File: tools/src/main/java/com/logistical/model/Staff.java
package com.logistical.model;

public class Staff {
  private String type;
  private String subType;
  private int number;

  public Staff(String type, String subType, int number, int price) {
    this.type = type;
    this.subType = subType;
    this.number = number;
    this.price = price;
  }

  private int price;

  public String getType() {
    return type;
  }

  public String getSubType() {
    return subType;
  }

  public int getNumber() {
    return number;
  }

  public int getPrice() {
    return price;
  }

}

File: tools/src/main/scala/com/logistical/model/Staff.scala
package com.logistical.model

/**
  * type 大类, subType 小类, number 数量, price 单价
  * 在构造器中需要直接提供这四个属性(String String int int)
  */
case class Staff(`type`: String, subType: String, number: Int, price: Int)
